deepchem/tailwind.config.js

```diff
@@ -0,0 +1,34 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
Can you add documentation to README about the css options that you are using?

Reply:
Added necessary documentation.